flask_wiki/static/css/wiki.css

```
@@ -47,22 +47,63 @@ h1 {
   color: #6c757d;
 }
 
-.wiki-page .toc ul {
+/* Article and table of contents: one column on a small screen, two on a wide
+   one. An application whose own header is fixed on top of the page declares its
+   height in `--wiki-sticky-offset`. */
+.wiki-page {
+  --wiki-toc-top: calc(var(--wiki-sticky-offset, 0px) + 1rem);
+  display: grid;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)